subprojects/internal-testing/src/main/groovy/org/gradle/integtests/fixtures/JUnitTestClassExecutionResult.groovy

```diff
@@ -145,7 +147,7 @@ class JUnitTestClassExecutionResult implements TestClassExecutionResult {
     }
 
     private NodeChild testCase(String name) {
-        testClassNode.testcase.find { it.@name == name }
+        testClassNode.testcase.find { it.@name == name || it.@name == "$name()"}
```

Comment:
❌ @marcphilipp said that there were quite a few issues mapping JUnit 5 to the old report format. But as far as I can see we don't have any tests that ensure a safe mapping. We are just reusing our existing tests and ignoring parentheses.

Reply:
As discussed on our call this morning, I won't hold up this PR because of this issue, but we should investigate this. I'm sure our users will tell us if something doesn't work well on some CI server, but if we can find the biggest issues before them that would be great :)